revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
     }
 }
 
+body {
+    font-family: Lato;
+}
+
+.row {
+    margin-right: 0px;
+    margin-left: 0px;
+    max-width: 100rem;
+
+    &.heading {
+        height: 500px;
+    }
+
+    &.about, &.project-updates, &.dashboard {
+        margin-right: 15%;
+        margin-left: 15%;
+        max-width: 70%;
+    }
+
+    &.dashboard {
+        height: 600px;
+    }
+
+    &.categories {
+        max-width: 100%;
+    }
+}
+
+.header-img-div {
+    width: 100%;
+    height: 500px;
+    position: absolute;
```

Comment:
Why does this need to be position: absolute? You're not setting bottom, top, left, or right